ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh

```diff
@@ -49,4 +49,12 @@ sosreport --ticket-number "\$HOSTNAME" --batch -o container_log,filesys,kvm,libv
 
 # Get libvirt logs
 tar -czC "/var/log/libvirt/qemu" -f "/tmp/artifacts/libvirt-logs-\$HOSTNAME.tar.gz" --transform "s?^\.?libvirt-logs-\$HOSTNAME?" .
+
+
+# Get the Botstrap logs if its still around, due to an error
+. common.sh
+. network.sh
+ssh -o 'ConnectTimeout=5' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -o 'ServerAliveInterval=90' core@\$BOOTSTRAP_PROVISIONING_IP \
+    sudo bash -c \\"ls /var/lib/containers/storage/overlay-containers/*/userdata/ctr.log /var/log/journal/*/system.journal \\| xargs -t tar -czf - \\" > /tmp/artifacts/bootstrap-\$HOSTNAME.tar.gz  || true
```

Comment:
There's an existing script on the bootstrap to collect more logging, I think that's more appropriate. It's /usr/local/bin/installer-gather.sh on the bootstrap.

Source: https://github.com/openshift/installer/blob/45fd42fd0827e7a259043d54027efb3df868cde4/data/data/bootstrap/files/usr/local/bin/installer-gather.sh

Reply:
thanks, will update